src/library/Uno.Material/Controls/Chips/ChipGroup.cs

```diff
@@ -0,0 +1,371 @@
+
+		#region DependencyProperty: SelectedItem
+
+		public static DependencyProperty SelectedItemProperty { get; } = DependencyProperty.Register(
+			nameof(SelectedItem),
+			typeof(object),
+			typeof(ChipGroup),
+			new PropertyMetadata(default, (s, e) => (s as ChipGroup)?.OnSelectedItemChanged(e)));
+
+		public object SelectedItem
+		{
+			get => (object)GetValue(SelectedItemProperty);
+			set => SetValue(SelectedItemProperty, value);
+		}
+
+		#endregion
+		#region DependencyProperty: SelectedItems
+
+		public static DependencyProperty SelectedItemsProperty { get; } = DependencyProperty.Register(
+			nameof(SelectedItems),
+			typeof(IList),
+			typeof(ChipGroup),
+			new PropertyMetadata(default, (s, e) => (s as ChipGroup)?.OnSelectedItemsChanged(e)));
+
+		public IList SelectedItems
+		{
+			get => (IList)GetValue(SelectedItemsProperty);
+			set => SetValue(SelectedItemsProperty, value);
+		}
+
+		#endregion
+		#region DependencyProperty: SelectionMemberPath
+
+		public static DependencyProperty SelectionMemberPathProperty { get; } = DependencyProperty.Register(
+			nameof(SelectionMemberPath),
+			typeof(string),
+			typeof(ChipGroup),
+			new PropertyMetadata(default, (s, e) => (s as ChipGroup)?.OnSelectionMemberPathChanged(e)));
+
+		/// <summary>
+		/// Gets or sets the name or path of the property that indicates selection state for each data item.
+		/// </summary>
+		public string SelectionMemberPath
+		{
+			get => (string)GetValue(SelectionMemberPathProperty);
+			set => SetValue(SelectionMemberPathProperty, value);
+		}
+
+		#endregion
+		#region DependencyProperty: SelectionMode = ChipSelectionMode.Single
+
+		public static DependencyProperty SelectionModeProperty { get; } = DependencyProperty.Register(
+			nameof(SelectionMode),
+			typeof(ChipSelectionMode),
+			typeof(ChipGroup),
+			new PropertyMetadata(ChipSelectionMode.Single, (s, e) => (s as ChipGroup)?.OnSelectionModeChanged(e)));
+
+		public ChipSelectionMode SelectionMode
+		{
+			get => (ChipSelectionMode)GetValue(SelectionModeProperty);
+			set => SetValue(SelectionModeProperty, value);
+		}
+
+		#endregion
+
+		private bool _isSynchronizingSelection = false;
+		private bool _isUpdatingSelection = false;
+
+		public ChipGroup()
+		{
+#if WINDOWS_UWP
+			RegisterPropertyChangedCallback(ItemsSourceProperty, (s, e) => (s as ChipGroup)?.OnItemsSourceChanged());
+#endif
+
+			this.Loaded += OnLoaded;
+		}
+
+		protected override void OnApplyTemplate()
+		{
+			base.OnApplyTemplate();
+		}
+
+		private void OnLoaded(object sender, RoutedEventArgs e)
+		{
+			EnforceSelectionMode();
+		}
+
+		private void OnSelectionMemberPathChanged(DependencyPropertyChangedEventArgs e)
+		{
+			var binding = SelectionMemberPath != null
+				? new Binding { Path = new PropertyPath(SelectionMemberPath), Mode = BindingMode.TwoWay }
+				: null;
+			foreach (var container in GetItemContainers())
+			{
+				if (binding != null) container.SetBinding(Chip.IsCheckedProperty, binding);
```

Comment:
I think this line is worth of curly brackets ^^

Reply:
I think this should be fine for one-liner, but okay.